app/domain/authentication/authn_jwt/jwt_mandatory_claim.rb

```diff
@@ -0,0 +1,15 @@
+module Authentication
+  module AuthnJwt
+
+    # This class instance holds a JWT standard claim which is
+    # mandatory to validate.
+    class JwtMandatoryClaim
```

Comment:
do we really need the `Mandatory`? will we have a `JwtOptionalClaim` class?

Reply:
I dont think so, but for me its more clear, do you prefer JwtClaim?

Reply:
yes. not a blocker though

Reply:
Change it due to the discussion above about the Mandatory messages